ansible/group_vars/all/defaults.yaml

```
@@ -5,17 +5,17 @@ python_path: ""
 # If it's not there, the kubernetes_full_version will have "None" for a version number.
 #
 # IMPORTANT When you update kubernetes_version, also update crictl_version.
-kubernetes_version: "1.29.6"
+kubernetes_version: "1.30.5"
 kubernetes_major_minor: "{{ (kubernetes_version.split('.') | map('trim'))[:2] | join('.') }}"
 kubernetes_semver: "v{{ kubernetes_version }}"
 
-kubernetes_cni_version: "1.3.0"
+kubernetes_cni_version: "1.5.1"
```

I think it should be like this, I'm curious if you saw `1.5.1` somewhere else?
```suggestion
kubernetes_cni_version: "1.4.0"
```

---

I did refer the below link based on https://github.com/mesosphere/konvoy-image-builder/blob/73172e0a5abc96ac936da338b3b94b6dad3ffd97/ansible/group_vars/all/defaults.yaml#L69

https://github.com/containernetworking/plugins/releases

---

@dkoshkin So do we have to change the cni version to 1.4.0 ?